layouts/_default/baseof.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
     {{- partial "head.html" . -}}
```

Comment:
This should be based on the language of the site defined in the configuration. I think the variable `.Site.Language.Lang` is the right one to use here.